app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great implementation with this submission however, the requirement requested that the application should have `:confirmable` enabled so users get to confirm their password. Kindly work on that :pleading_face: 

![image](https://github.com/JuanDiegoGuerra/Blog-App-RoR/assets/99715640/67b287c7-53e2-42a2-a14d-89afa5b603e1)